### PR TITLE
kernel: Fix ExecCounter

### PR DESCRIPTION
**Pull request by gaocegege**
Signed-off-by: Ce Gao <gaoce@caicloud.io>

**What this PR does / why we need it**:

Supported exec counting.

**Special notes for your reviewer**:

/cc @ddysher 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by gaocegege** [comment]
Close #32

File: pkg/kernel/kernel.go [hunk 1]
```diff
@@ -284,7 +287,7 @@ func (k Kernel) doEval(code string) (val []interface{}, err error) {
 	job, err := k.Manager.Execute(code)
 	if err != nil {
 		fmt.Printf("Failed to create job: %v", err)
-		return nil, err
+		return nil, nil
```
Comment (ddysher) [inline pkg/kernel/kernel.go]: why returning nil here?

File: pkg/kernel/kernel.go [hunk 2]
```diff
@@ -21,16 +21,20 @@ const (
 	kernelIdle     = "idle"
 )
 
+var (
+	// ExecCounter is incremented each time we run user code in the notebook.
+	// It cannot be defined in Kernel, or it will be always 1.
```
Comment (ddysher) [inline pkg/kernel/kernel.go]: This is a little misleading without any context, maybe rephrase it to `It cannot be defined in Kernel struct`?

Comment (ddysher) [reply pkg/kernel/kernel.go]: btw, what's the concrete use case of exec counter? is it required by jupyter?

File: pkg/kernel/kernel.go (hunk 1 above)
Comment (gaocegege) [reply pkg/kernel/kernel.go]: To keep consistency with golang kernel, and for the execcounter. We have already logged the error.

File: pkg/kernel/kernel.go (hunk 2 above)
Comment (gaocegege) [reply pkg/kernel/kernel.go]: Yeah, it is required by Jupyter
